### ISSUE TITLE
Client crashes after connecting to server

### ISSUE THREAD

**Issue by Ryhon0** [alvr-org/ALVR#894]
## Description
After I connect to a server the client just crashes and server logs this message: `Client disconnected. Cause: At alvr/common/src/sockets/control_socket.rs:32`

## General Troubleshooting
- [x] I carefully followed the instructions in the [README](https://github.com/alvr-org/ALVR/blob/master/README.md) and successfully completed the setup wizard
- [x] I read the ALVR Wikis [here](https://github.com/polygraphene/ALVR/wiki) and [here](https://github.com/alvr-org/ALVR/wiki)

### Hardware
hw-probe: https://linux-hardware.org/?probe=5b7f950dfe

### Installation
**ALVR Version**:
v16.0.0

**SteamVR Version**:
beta 1.21.4, also tried whatever the non-beta version is

**Install Type**:
Server: Installed `alvr` from the AUR
Client: tried both the GitHub release and the release on SideQuest

**OS Name and Version**
Arch Linux

**Comment by GrArmpit** [comment]
I had the same issue with the added error message ` failed to create VAAPI encoder`. If I disable `Steam Game Audio` in the ALVR Server Settings it no longer crashes, it simply hangs on `The stream will begin soon, Please wait.` It cures the `Client disconnected. Cause: At alvr/common/src/sockets/control_socket.rs:32` error, but the VAAPI error persists. I'm not sure if that's progress...

**Comment by GrArmpit** [comment]
I got it working!
Steps taken to fix:
Install Ungoogled Chromium (Presumably any chromium based browser would work for this)
Use https://www.linuxuprising.com/2021/01/how-to-enable-hardware-accelerated.html as a guideline to enable VAAPI in the browser
Add an environment variable `ALCRO_BROWSER_PATH` targeting the newly flagged .desktop file, as suggested under the installation page.

At this point, the ALVR Server UI stopped appearing, so I opened `localhost:8082` in Ungoogled Chromium as per issue #892, and this connected and brought me to SteamVR Home. The VAAPI error still shows, but it runs so I guess its fine?
In any case, hope this works for you.
